build/ferretdb.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=FerretDB Database
```

Comment:
`Description=FerretDB` or if we want  consistent description across different builds? See nixOS https://github.com/NixOS/nixpkgs/blob/nixos-unstable/pkgs/servers/nosql/ferretdb/default.nix#L41
`Description=Truly Open Source MongoDB alternative`

Reply:
https://www.freedesktop.org/software/systemd/man/latest/systemd.unit.html#Description=
> This string also shouldn't just repeat the unit name

https://nixos.org/manual/nixpkgs/stable/#var-meta-description
> A library for decoding PNG images

So while both fields are called `description`, they mean different things. I don't think we should make them the same just because of the field name